### PR TITLE
fix: button should accept hoverFg

### PR DESCRIPTION
**Pull request by P0lip**

### CODE REVIEW

File: src/Button.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ export const buttonStyles = (theme: ITheme['button'], { disabled }: IButton = {}
 
       ':hover': {
         backgroundColor: theme.hoverBg,
+        ...(theme.hoverFg && { color: theme.hoverFg }),
```
Comment (casserni) [inline src/Button.tsx]: why need the `...` while background doesnt?

Comment (P0lip) [reply src/Button.tsx]: It's optional

Comment (P0lip) [reply src/Button.tsx]: Could also be
```color: theme.hoverFg || themefg```

Comment (marbemac) [reply src/Button.tsx]: will css not figure it out if it gets an undefined value?

Comment (P0lip) [reply src/Button.tsx]: @marbemac, it does, nice.
Code updated.

**Comment by P0lip** [comment]
> Looks good. However it would be nice to understand when a property is mandatory or it's optional

True! Right now we tend to mix it - it's all about common sense at the moment (unlike background you usually don't change hoverFg for buttons for hove state etc.), but it might be good to have some enforced rules.

**Pull request merged**